mysql/update_statement_test.go

```diff
@@ -6,6 +6,21 @@ import (
 	"testing"
 )
 
```

Comment:
Could you add at least one integration test in `/tests/mysql`.

Reply:
Added!